resources/views/marital_status.blade.php

```diff
@@ -0,0 +1,74 @@
+@extends('layouts.app')
+@extends('layouts.nav')
+
+@section('title', 'MaritalStatus')
+
+@section('nav')
+@endsection
+
+@section('content')
+<div class="chart">
+  <!-- chart.js -->
+  <canvas id="pieCanvas"></canvas>
+  <!-- end -->
+</div>
+
+<script>
+  var ctx = document.getElementById("pieCanvas");
+  ctx.style.height = 750;
+  var pieCanvas = new Chart(ctx, {
+    // kind of grapheme_strpos
+    type: 'doughnut',
```

Comment:
It might be good to use 'pie chart'.
However, it's ok to use 'doughnut chart', I think.
Let's talk about that later.